CHANGELOG.md

```diff
@@ -84,13 +92,43 @@ This release made possible by the awesome work of the following contributors (in
 
 <3
 
-### Support ###
+## 0.3
+
+### [0.3.1] - 2017-05-19
```

Comment:
I did write change logs for a bunch of these, at https://github.com/jupyterhub/helm-chart/releases!